client/src/main/java/com/distkv/dst/client/DstSortedListProxy.java

```diff
@@ -94,15 +93,15 @@ public void putMember(String key, SortedListEntity entity) {
     CheckStatusUtil.checkStatus(response.getStatus(), key, typeCode);
   }
 
-  public List<Integer> getMember(String key, String member) {
+  public LeaderboardItem getMember(String key, String member) {
     SortedListProtocol.GetMemberRequest.Builder getMemberRequest =
-            SortedListProtocol.GetMemberRequest.newBuilder();
+        SortedListProtocol.GetMemberRequest.newBuilder();
```

Comment:
I don't modify the place. But It shows that I changed. There may be github bug.

Reply:
No. It might be that you did this by IDE